platform/core/src/studies/retrieveStudiesMetadata.js

```diff
@@ -13,19 +13,15 @@ import { retrieveStudyMetadata } from './retrieveStudyMetadata';
 export default function retrieveStudiesMetadata(
   server,
   studyInstanceUids,
-  seriesInstanceUids
+  filters
```

Comment:
Can we destructure this at some point, so it's clearer what possible filters can be?

Reply:
ok